service.go

```diff
@@ -88,6 +88,23 @@ const (
 	optionLaunchdConfig = "LaunchdConfig"
 )
 
+// Status represents service status as an interger value
+type Status int
+
+// Status of service represented as an integer
+const (
+	StatusNotImplemented Status = iota
```

Comment:
It doesn't look like this status is used. Remove?

Reply:
This is leftover from before I implemented service status checks for all Systems. It can probably go now